tests/e2e/e2e_common.go

```diff
@@ -39,6 +40,7 @@ const (
 	defaultPandoraSyncWaitTime                 = 90
 	defaultVCRebootWaitTime                    = 180
 	destinationDatastoreURL                    = "DESTINATION_VSPHERE_DATASTORE_URL"
+	disklibUnlikErr                            = "DiskLib_Unlink"
```

Comment:
nit: disklibUnlikErr -> disklibUnlinkErr

Reply:
Fixed it.